pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2681,3 +2688,13 @@ func resetCachedListenerConfig(mesh *meshconfig.MeshConfig) {
 func listenerKey(bind string, port int) string {
 	return bind + ":" + strconv.Itoa(port)
 }
+
+func dropAlpnFromList(alpnProtocols []string, alpnToDrop string) []string {
+	newAlpnProtocols := []string{}
+	for _, alpn := range alpnProtocols {
+		if alpn != alpnToDrop {
+			newAlpnProtocols = append(newAlpnProtocols, alpn)
+		}
+	}
```

Comment:
Making this suggestion as per Mat Ryer's line of sight principle: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 
```suggestion
	for _, alpn := range alpnProtocols {
		if alpn == alpnToDrop {
			continue
		}
		newAlpnProtocols = append(newAlpnProtocols, alpn)
	}
```